core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -247,6 +260,9 @@ private void deactivateInterruptCycle() {
    * Scheduled routine that interrupts waiters/holders of shared lock.
    */
   private void waiterInterruptRoutine() {
+    LOG.debug("Interrupt-cycle interrupting %d waiters/holders: {%s}",
+        Thread.currentThread().getId(), mSharedWaitersAndHolders.size(), mSharedWaitersAndHolders
+            .stream().map((th) -> Long.toString(th.getId())).collect(Collectors.joining(",")));
```

Comment:
I think that we might want to wrap this in a `if (Log.isDebugEnabled())` because grabbing iterating, and joining into a String can be pretty expensive?

Reply:
I made it INFO now. We don't want to miss this trace when it happens.

Reply:
How often do we expect this method to run?

Reply:
every 30secs by default only if it is activated under certain conditions.

Reply:
ok, I think it's fine then. I just wanted to make sure it doesn't need to get computed frequently.